package.json

```diff
@@ -41,6 +41,7 @@
     "rollup-plugin-typescript2": "^0.30.0",
     "ts-jest": "^26.4.4",
     "tslib": "^2.1.0",
-    "typescript": "^4.2.3"
+    "typescript": "^4.2.3",
+    "yarn-audit-ci": "^1.2.0"
```

Comment:
i'd persoanly like to keep it tthan re-implement the same ssimple logic to stop the build if critcal vulns are found. i dont think it's been updated since it's simple functionality